packages/client/src/components/molecules/LogoStatusMolecules/Settings/SetFinishLineLocation.tsx

```
@@ -0,0 +1,102 @@
+import { CoordUpdateResponse } from "@shared/helios-types";
+
+const SetFinishLineLocation = () => {
+  const { currentPacket } = usePacketStore();
+  const { resolvedTheme } = useTheme();
+  const [password, setPassword] = React.useState("");
+
+  const handleSetFinishLineLocation = () => {
+    socketIO.emit(
+      "setLapCoords",
+      {
+        lat: (currentPacket?.Telemetry.GpsLatitude).toString() || "0",
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)